dips/dip-196.md

```diff
@@ -0,0 +1,70 @@
+---
+dip: 196
+title: Onchain Network Identity
+authors: Greg Nazario (@gregnazario)
+status: Draft
+type: Informational
+created: 07/06/2021
+last_updated: 07/06/2021
+issue: https://github.com/diem/dip/issues/196
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Simple Summary
```

Comment:
Probably no need to have summary, abstract, and motivation, maybe just one section?

Reply:
We should update DIP-0 appropriately.